assets/css/map/markers/_location_dot_icon.scss

```diff
@@ -0,0 +1,17 @@
+.c-location-dot-icon {
+  svg {
+    overflow: visible;
+    width: 18px !important;
+    height: 24px !important;
```

Comment:
question(non-blocking): is `width` and `height` necessary if it's specified by the leaflet divIcon?

Reply:
Ah, yup, good catch. Leaflet sets the height / width of the element inline so this is redundant.